static/compatibilities/cert-manager.yaml

```diff
@@ -2,6 +2,10 @@ icon: https://raw.githubusercontent.com/cert-manager/cert-manager/d53c0b9270f8cd
 git_url: https://github.com/cert-manager/cert-manager
 release_url: https://github.com/cert-manager/cert-manager/releases/tag/v{vsn}
 versions:
+- version: 1.15.0
+  kube: ['1.24', '1.25', '1.26', '1.27', '1.28', '1.29', '1.30']
```

Comment:
why is this one in ascending order while others are descending?

Reply:
Hmm, I'm not sure; that's definitely weird. I'll look into it.